src/main/java/lotto/domain/WonLotto.java

```diff
@@ -0,0 +1,49 @@
+package lotto.domain;
+
+public class WonLotto {
+
+    private Lotto wonLotto;
+    private int bonusNo;
+
+    private LottoGenerator lottoGenerator = new LottoGenerator();
+
+    private final int NON_GRADE_NUMBER = 3;
+    private final int BONUS_NUMBER_MATCHING_COUNT = 5;
+
+    public WonLotto(){
+        this.wonLotto = lottoGenerator.generateLotto();
+        this.bonusNo = lottoGenerator.generateBonus(wonLotto);
+    }
+
+    public WonLotto(Lotto lotto, int bonusNo){
+        this.wonLotto = lotto;
+        this.bonusNo = bonusNo;
+    }
+
+    public Lotto getWonLotto(){
+        return wonLotto;
+    }
+
+    public int getBonusNo(){
+        return bonusNo;
+    }
+
+    public LottoRank checkRanking(Lotto lotto){
+        int matchNo=wonLotto.checkSameCount(lotto);
+
+        if(matchNo < NON_GRADE_NUMBER)
+            return LottoRank.NONE;
+
+        if(matchNo == BONUS_NUMBER_MATCHING_COUNT)
+            return lotto.getLotto().contains(bonusNo) ? LottoRank.SECOND : LottoRank.THIRD;
+
+        if(matchNo < BONUS_NUMBER_MATCHING_COUNT)
+            return LottoRank.values()[convertRank(matchNo)];
+
+        return LottoRank.FIRST;
```

Comment:
`LottoRank` 생성에 대한 책임을 `WonLotto`에서 해주고 있는 것으로 보여요.
결과적으로 `WonLotto`에서 `LottoRank`의 각 단계별 매칭 조건을 다 알고 있어야 할것 같습니다.
매칭에 필요한 정보를 `LottoRank`에게 줘서 적절한  `LottoRank`를 받아오는 방법도 있을것 같아요
예를 들면,
```suggestion
        int matchNo=wonLotto.checkSameCount(lotto);
        boolean isBonusMatched=lotto.getLotto().contains(bonusNo);
        
        return LottoRank.rankForMatched(matchNo, isBonusMatched);
```